style.css

```diff
@@ -0,0 +1,361 @@
+    min-width: 1.5rem;
+    max-width: fit-content;
+    height: 1.5rem;
+    background: var(--badge-color);
+    color: var(--light-color);
+    border-radius: 2rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0.2rem;
+}
+
+/* Image Slider */
+
+.img-slider {
+    width: 100%;
+    height: 75vh;
+    position: relative;
+    overflow: hidden;
+    box-shadow: rgba(14, 30, 37, 0.12) 0px 2px 4px 0px, rgba(14, 30, 37, 0.32) 0px 2px 16px 0px, rgba(0, 0, 0, 0.35) 0px -50px 36px -28px inset, rgba(50, 50, 93, 0.25) 0px 30px 60px -12px inset, rgba(0, 0, 0, 0.3) 0px 18px 36px -18px inset;
+}
+
+.img-slider-item {
+    position: absolute;
+    inset: 0;
+}
+
+.btn-round {
+    position: absolute;
+    top: 50%;
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
+    cursor: pointer;
+    outline: none;
+    border: none;
+    background: rgba(0, 0, 0, 0.5);
+    color: white;
+    transition: var(--default-transition);
+    text-align: center;
+}
+
+.btn-round:hover {
+    transform: scale(1.1);
+}
+
+.previous {
+    left: 1rem;
+}
+
+.next {
+    right: 1rem;
+}
+
+.dots {
+    position: absolute;
+    bottom: 1rem;
+    left: 50%;
+    transform: translate(-50%);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 0.75rem;
+}
+
+.dot {
+    width: 1rem;
+    height: 1rem;
+    border-radius: 50%;
+    background: rgba(0, 0, 0, 0.8);
+    border: 2px solid white;
+    cursor: pointer;
+    transition: all 0.2s ease-in;
+}
+
+.active,
+.dot:hover {
+    transform: scale(1.3);
+    background: rgba(255, 255, 255, 0.8);
+    border: 2px solid black;
+}
+
+/* Main */
+.main {
+    margin-top: 2rem;
+    padding: 2rem;
+}
+
+.categories {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 1rem;
+}
+
+.category {
+    position: relative;
+    padding: 1rem 1.5rem;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    gap: 1.5rem;
+    border-radius: 2rem;
+    box-shadow: rgba(50, 50, 93, 0.25) 0px 30px 60px -12px inset, rgba(0, 0, 0, 0.3) 0px 18px 36px -18px inset;
+}
+
+.category > h1,
+.category > a {
+    z-index: 2;
+}
+
+.fader {
+    position: absolute;
+    inset: 0;
+    z-index: 1;
+    background: rgba(0, 0, 0, 0.5);
+    border-radius: 2rem;
+}
+
+.sports-eqp {
+    background: url('./resources/sportsimg.jpg') center;
+    color: var(--white-color)
+}
+
+.sports-wear {
+    background: url('./resources/sportswear.jpg') center;
+    color: var(--white-color)
+}
+
+.games {
+    background: url('./resources/gaming.jpg') center;
+    color: var(--white-color)
+}
+
+.accessories {
+    background: url('./resources/gamingaccessories.jpg') center;
+    color: var(--white-color)
+}
+
+.covers {
+    background: url('./resources/mobilecovers.jpg') center;
+    color: var(--white-color)
+}
+
+.merch {
+    background: url('./resources/merchandise.jpg') center;
+    color: var(--white-color)
+}
+
+.btn-to {
```

Comment:
what does `btn-to` mean? `to` abbreviates something?

Reply:
Yeah, I named it to buttons that take you to that related product listing page.